librz/il/core_theory_opcodes.h

```diff
@@ -0,0 +1,169 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZIL_CORE_THEORY_DEFINITIONS_H
+#include "_ct_opcodes.h"
+
+typedef enum {
+	// Init
+	RZIL_OP_VAR,
+	RZIL_OP_UNK,
+	RZIL_OP_ITE,
+
+	// Bool
+	RZIL_OP_B0,
+	RZIL_OP_B1,
+	RZIL_OP_INV,
+	RZIL_OP_AND_,
+	RZIL_OP_OR_,
+
+	// BitVector
+	RZIL_OP_INT,
+	RZIL_OP_MSB,
+	RZIL_OP_LSB,
+	RZIL_OP_NEG,
+	RZIL_OP_NOT,
+	RZIL_OP_ADD,
+	RZIL_OP_SUB,
+	RZIL_OP_MUL,
+	RZIL_OP_DIV,
+	RZIL_OP_SDIV,
+	RZIL_OP_MOD,
+	RZIL_OP_SMOD,
+	RZIL_OP_LOGAND,
+	RZIL_OP_LOGOR,
+	RZIL_OP_LOGXOR,
+	RZIL_OP_SHIFTR,
+	RZIL_OP_SHIFTL,
+	RZIL_OP_SLE,
+	RZIL_OP_ULE,
+	RZIL_OP_CAST,
+	RZIL_OP_CONCAT,
+	RZIL_OP_APPEND,
+	// ...
+
+	// Memory
+	RZIL_OP_LOAD,
+	RZIL_OP_STORE,
+
+	// Effects (opcode with side effects)
+	RZIL_OP_PERFORM,
+	RZIL_OP_SET,
+	RZIL_OP_JMP,
+	RZIL_OP_GOTO,
+	RZIL_OP_SEQ,
+	RZIL_OP_BLK,
+	RZIL_OP_REPEAT,
+	RZIL_OP_BRANCH,
+
+	RZIL_OP_INVALID
+} CoreTheoryOPCode;
```

Comment:
```suggestion
} RzILOPCode;
```

Is it necessary to have `CoreTheory` in the name? cc @XVilka

Reply:
No, see my comment before:
> We should choose either to use "core theory" everywhere or "rzil". Except for comments explaining the origin of the RZIL from the Core Theory, and the correspondence of the operations, I think we should use "rzil" or "RZIL" everywhere.